app/javascript/components/enrollment/steps/Identification.js

```diff
@@ -304,6 +304,7 @@ class Identification extends React.Component {
                   <Form.Label htmlFor="workflow-select" className="input-label">
                     WORKFLOW *
                   </Form.Label>
+                  {this.props.edit_mode && <InfoTooltip tooltipTextKey="editingWorkflow" location="right"></InfoTooltip>}
```

Comment:
Is there a reason we can't put the tooltip on the disabled field instead of an info tooltip? that was it is directly related to the input when you hover over the disable field

Reply:
Updated in https://github.com/SaraAlert/SaraAlert/pull/1118/commits/60525138ccead3a44bdd28fc224c25418e209d99.